<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="da940fd0-1d5d-451a-b2d7-1421de2903ed">
      <driver-ref>mysql</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/autotest/settings.py</remarks>
      <jdbc-driver>com.mysql.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://127.0.0.1:3306/autotest</jdbc-url>
      <driver-properties>
        <property name="autoReconnect" value="true" />
        <property name="yearIsDateType" value="false" />
        <property name="characterEncoding" value="utf8" />
        <property name="zeroDateTimeBehavior" value="convertToNull" />
        <property name="characterSetResults" value="utf8" />
        <property name="tinyInt1isBit" value="false" />
      </driver-properties>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="b031d025-0db5-4aab-8a13-e78f27a212b5">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/../guest/guest/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\guest\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
  </component>
</project>